aries-site/src/examples/components/maskedinput/MaskedSizeUnitsExample.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { Form, FormField, MaskedInput } from 'grommet';
+
+export const MaskedSizeUnitsExample = () => {
+  const [value, setValue] = useState('');
+
+  return (
+    <Form>
+      <FormField
+        label="Size"
+        htmlFor="masked-size-units"
+        name="maskedSizeUnits"
+      >
+        <MaskedInput
+          name="maskedSizeUnits"
+          id="masked-size-units"
+          mask={[
```

Comment:
Suggestion: separate the mask logic from the MaskInput to aid readability and cognition.